cmd/flightctl-alert-exporter/main.go

```diff
@@ -21,14 +25,41 @@ import (
 )
 
 func main() {
-	ctx := context.Background()
-
 	log := log.InitLogs()
-	log.Println("Starting alert exporter")
+
+	if err := runCmd(log); err != nil {
+		log.Fatalf("Alert exporter error: %v", err)
+	}
+}
+
+func runCmd(log *logrus.Logger) error {
+	log.Info("Starting alert exporter")
+	defer log.Info("Alert exporter stopped")
+
+	// Single context with signal handling - OS signal cancels context
+	ctx, cancel := signal.NotifyContext(context.Background(),
+		syscall.SIGTERM, syscall.SIGINT, syscall.SIGQUIT)
```

Comment:
if we are cancelling the context  on signal arent we forcing the servers to exit abruptly is that graceful? i would of expected that this external parent signal handling would ensure shutdown (ctx cancel) after a global timeout. then internal to each server they would handle graceful shutdown directly.